ssntp/server.go

```diff
@@ -246,6 +247,15 @@ func (server *Server) Serve(config *Config, ntf ServerNotifier) error {
 	transport := config.transport()
 	uri = config.URI
 
+	payload, err := configuration.GetConfiguration(config.ConfigURI)
+
+	if err != nil {
+		server.log.Errorf("Error loading configuration: %s", err)
+		return err
```

Comment:
@mrkz There are some stuff in the configuration data that we can't guess or even build a smart default for, so yes it's ok for now.